og.og_permissions.yml

```diff
@@ -2,10 +2,11 @@ update group:
   title: 'Edit group'
   description: 'Edit the group. Note: This permission controls only node entity type groups.'
   'default role':
-    - OG_ADMINISTRATOR_ROLE
```

Comment:
From other PR:

> We could make these strings 'special'. A bit like VERSION is now for info yml files. Does this even work in it's current form? Doesn't the YAML parser just interpret that as a string?

We could still use the constants but we would need to add special handling for them as per above comment.